src/_codux/boards/add-note-card/add-note-card.board.tsx

```diff
@@ -0,0 +1,13 @@
+import { createBoard } from '@wixc3/react-board';
+import { AddNoteCard } from '../../../components/add-note-card/add-note-card';
+import { ComponentWrapper } from '../../board-wrappers/component-wrapper';
+
+export default createBoard({
+    name: 'AddNoteCard',
+    Board: () => (
+        <ComponentWrapper>
```

Comment:
What is `ComponentWrapper`?

Reply:
Can we rename it to a FakeRouter or something like this?

Reply:
Agree. Maybe also add a component template with a fake router?

Reply:
And another point here. You need to wrap a part of a markup, with `ContentSlot` (part of '@wixc3/react-board'), to omit such fake wrapper when using it via AddPanel

Reply:
> Agree. Maybe also add a component template with a fake router?

Sounds good. But i'm not sure it's a component or board template